packages/react-dom/src/client/ReactInputSelection.js

```diff
@@ -82,7 +83,7 @@ export function restoreSelection(priorSelectionInformation) {
       }
     }
 
-    priorFocusedElem.focus();
+    focusNode(priorFocusedElem);
```

Comment:
Let’s just inline the try/catch here and leave a comment that this is for SVG in IE11.